anyway/widgets/suburban_widgets/accident_count_pedestrians_per_vehicle_street_vs_all_widget.py

```diff
@@ -39,3 +34,10 @@ def accidents_count_pedestrians_per_vehicle_street_vs_all_mock_data():  # Tempor
             {"location": "בן יהודה", "vehicle": "רכב כבד", "num_of_accidents": 22},
             {"location": "בן יהודה", "vehicle": "אופניים וקורקינט ממונע", "num_of_accidents": 9},
         ]
+
+    @staticmethod
+    def localize_items(request_params: RequestParams, items: Dict) -> Dict:
+        items["data"]["text"] = {
+            "title": "Number of pedestrian accidents on Ben Yehuda street in Tel Aviv by type of hitting vehicle compared to urban accidents across the country"

```

Comment:
No translation here?

Reply:
no, on purpose, since this is hard-coded mock data. should i add?